okapi-core/src/main/java/org/folio/okapi/util/ProxyContext.java

```diff
@@ -164,7 +164,7 @@ public RoutingContext getCtx() {
     return ctx;
   }
 
-  private String getReqId() {
+  private String  getReqId() {
```

Comment:
There is an extra space before `getReqId()`

Reply:
done